README.md

```diff
@@ -1026,6 +1026,18 @@ All statements above will return false if used with `===`
 
 <p align="right"><a href="#table-of-contents">⬆ Return to top</a></p>
 
+## ![✔] 6.26. Have a SECURITY.TXT File [PRODUCTION]
```

Comment:
Seeing https://securitytxt.org/ I understand that the filename should be in lowercase.

They also say that the preferred location is under the .well_known directory.

Additionally I think that we should include the link for further reference.

What do you think of that?

Reply:
Agree with @kevynb . Will update.

Reply:
Updated the file with references. @kevynb Can be merged